src/build/_compat.py

```diff
@@ -25,18 +23,4 @@ def wrapper(cls):  # type: (_T) -> _T
     return wrapper
 
 
-if sys.version_info[0] == 2:
-    abstractproperty = abc.abstractproperty
-else:
-    from typing import Any, cast
-
-    F = TypeVar('F', bound=Callable[..., Any])
-
-    def abstractproperty(func):  # type: (F) -> F
-        return cast(F, property(abc.abstractmethod(func)))
-
-
-__all__ = (
-    'abstractproperty',
-    'add_metaclass',
-)
+__all__ = ('add_metaclass',)
```

Comment:
Please restore this. mypy was happy on the CI and for me locally.

Reply:
It wasn't with the changes, probably a mypy bug.